core/src/global/global.css

```diff
@@ -129,6 +129,25 @@ th {
 	text-align: left;
 }
 
+legend {
+	display: table;
+	/* Correct `color` not being inherited in IE 8/9/10/11 */
+	color: inherit;
+	/* Remove padding so people aren't caught out if they zero out fieldsets */
+	padding: 0;
+}
```

Comment:
Hi, if you don't mind, may I ask where does this come from?

Reply:
Hi, I got from this [normalize.css ](https://github.com/necolas/normalize.css/blob/master/normalize.css#L240)

Reply:
hm we did use modern-normalize.. not sure why they drop this..